src/libxtp/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ endif()
 #PROJECT_BINARY_DIR/include for votca_config.h
 target_include_directories(votca_xtp PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
   PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include> ${HDF5_INCLUDE_DIRS}
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include> ${SPECTRA_INCLUDE_DIRS}
```

Comment:
Why do you need $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include> again? And `${SPECTRA_INCLUDE_DIRS}` should go under `PRIVATE`.